src/configs/env.ts

```diff
@@ -536,14 +536,13 @@ export const SUPERSET_IRS_MOPUP_REPORTING_JURISDICTIONS_FOCUS_AREA_LEVEL =
   process.env.REACT_APP_SUPERSET_IRS_MOPUP_REPORTING_JURISDICTIONS_FOCUS_AREA_LEVEL ?? '0';
 export type SUPERSET_IRS_MOPUP_REPORTING_JURISDICTIONS_FOCUS_AREA_LEVEL = typeof SUPERSET_IRS_MOPUP_REPORTING_JURISDICTIONS_FOCUS_AREA_LEVEL;
 
-export const ENABLE_IRS_LITE_MOPUP_REPORTING =
-  process.env.REACT_APP_ENABLE_IRS_LITE_MOPUP_REPORTING;
-export type ENABLE_IRS_LITE_MOPUP_REPORTING = typeof ENABLE_IRS_LITE_MOPUP_REPORTING;
+export const SUPERSET_JURISDICTION_EVENTS_SLICE =
+  process.env.REACT_APP_SUPERSET_JURISDICTION_EVENTS_SLICE ?? '0';
+export type SUPERSET_JURISDICTION_EVENTS_SLICE = typeof SUPERSET_JURISDICTION_EVENTS_SLICE;
 
-export const SUPERSET_IRS_LITE_MOPUP_REPORTING_JURISDICTIONS_DATA_SLICES =
-  process.env.REACT_APP_SUPERSET_IRS_LITE_MOPUP_REPORTING_JURISDICTIONS_DATA_SLICES ?? '0';
-export type SUPERSET_IRS_LITE_MOPUP_REPORTING_JURISDICTIONS_DATA_SLICES = typeof SUPERSET_IRS_LITE_MOPUP_REPORTING_JURISDICTIONS_DATA_SLICES;
-
-export const SUPERSET_IRS_LITE_MOPUP_REPORTING_JURISDICTIONS_FOCUS_AREA_LEVEL =
-  process.env.REACT_APP_SUPERSET_IRS_LITE_MOPUP_REPORTING_JURISDICTIONS_FOCUS_AREA_LEVEL ?? '0';
-export type SUPERSET_IRS_LITE_MOPUP_REPORTING_JURISDICTIONS_FOCUS_AREA_LEVEL = typeof SUPERSET_IRS_LITE_MOPUP_REPORTING_JURISDICTIONS_FOCUS_AREA_LEVEL;
+export const ASSIGN_TEAMS_PLAN_TYPES_DISPLAYED = (
+  process.env.REACT_APP_ASSIGN_TEAMS_PLAN_TYPES_DISPLAYED ||
+  process.env.REACT_APP_DISPLAYED_PLAN_TYPES ||
+  'FI,IRS,MDA,MDA-Point,Dynamic-FI,Dynamic-IRS,Dynamic-MDA, IRS-Lite'
```

Comment:
Lets remove the whitespace before IRS-Lite

Reply:
done

Reply:
This was because this is being evaluated by splitting on comma so the whitespace would affect things and we'd end up with a value that has an unexpected space and wouldn't match the plan intervention type

Reply:
Gotcha. It's fixed now so shouldn't be a problem.